api/src/main/java/uk/gov/hmcts/fees2/register/util/FeeFactory.java

```diff
@@ -1,14 +1,13 @@
 package uk.gov.hmcts.fees2.register.util;
 
-import uk.gov.hmcts.fees2.register.api.contract.loader.request.LoaderFixedFeeDto;
 import uk.gov.hmcts.fees2.register.api.contract.request.*;
 import uk.gov.hmcts.fees2.register.data.model.*;
 
 public class FeeFactory {
 
     public static Fee getFee(FeeDto request){
         Fee fee = null;
-        if (request instanceof FixedFeeDto || request instanceof LoaderFixedFeeDto)
+        if (request instanceof FixedFeeDto)
```

Comment:
Please add a TODO here to fix Factory method. It shouldn't be using the instanceof